back-end/src/routes/users.ts

```diff
@@ -0,0 +1,51 @@
+import { Request, Router, Response } from "express";
+import fetch from "node-fetch";
+
+const queries = require('../db/mongodb').queries;
+const router = Router();
+console.log(queries);
+
+
+
+router.post("/", async (req: Request, res: Response) => {
+    try{
+    let id = await queries.addUser({name:req.body.name, email:req.body.email, password:req.body.password});
+    res.send(id);
+    }catch (error:any) {
+      console.error(error.message);
+      res.status(500).send('Server Error');
+    }
+  });
+  
+  router.get('/:user_id', async (req, res) => {
+    const userID = req.params.user_id;
+    try {
+    let user = await queries.getUser(userID);
+      res.send(user);
+    } catch (error:any) {
+      console.error(error.message);
+      res.status(500).send('Server Error');
+    }
+  });
+  
+  router.post('/login/', async (req, res) => {
```

Comment:
shouldn't the endpoint be /login not /login/, idk if it makes a diference though